src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -127,9 +137,10 @@ def _leaf_from_dict(description, config):
     display_type = description.pop("display_type", None)
     try:
         leaf_object = leaf_type(**description, config=config)
-    except KeyError:
+    except KeyError as exc:
         # If the strategy is not supported in SCM or normal operating mode, do
         # not create the area at all.
+        logging.error("Failed to create leaf %s %s %s", leaf_type, description, exc)
```

Comment:
Thanks a lot! I was about to ad this in one of my last PRs, but forgot. Took me a long time to find this spot. Debugging of the init of the simulation will be easier now.

Reply:
Although, let's follow the approach to first call 'getLogger' and do not use `logging` directly, ok?

Reply:
Agreed! Will fix this